concordium-consensus/src/Concordium/GlobalState/Persistent/Account.hs

```diff
@@ -359,16 +359,15 @@ instance forall m av. (MonadBlobStore m, IsAccountVersion av) => BlobStorable m
         su0 :: PersistentAccountStake 'AccountV0 -> m (Put, PersistentAccountStake 'AccountV0)
         su0 pas@PersistentAccountStakeNone = return (put (refNull :: BlobRef (PersistentAccountBaker av)), pas)
         su0 (PersistentAccountStakeBaker bkrref) = do
-          (r, bkrref') <- storeUpdate bkrref
+          (!r, !bkrref') <- storeUpdate bkrref
```

Comment:
Is this based on profiling, or just a guess? We have many other instances like this without forcing evaluation. Should those also be updated?

Reply:
I think this will not be helpful in this case, so I've reverted.